redash/query_runner/google_spreadsheets.py

```diff
@@ -166,11 +165,7 @@ def _get_spreadsheet_service(self):
         key = json_loads(b64decode(self.configuration['jsonKeyFile']))
         creds = ServiceAccountCredentials.from_json_keyfile_dict(key, scope)
 
-        timeout_session = HTTPSession()
-        timeout_session.requests_session = TimeoutSession()
-        spreadsheetservice = gspread.Client(auth=creds, http_session=timeout_session)
-        spreadsheetservice.login()
-        return spreadsheetservice
+        return gspread.authorize(creds)
```

Comment:
Why did you remove the timeout handling?

Reply:
oops. Because gspread.httpsession was not found, but I will fix it 
```
Traceback (most recent call last):
  File "main.py", line 2, in <module>
    from gspread.httpsession import HTTPSession
ModuleNotFoundError: No module named 'gspread.httpsession
```

Reply:
I fixed it 🙏 
https://gspread.readthedocs.io/en/latest/api.html?highlight=session#client